src/__tests__/components/fields/image-upload/image-upload.spec.tsx

```diff
@@ -134,6 +146,11 @@ const renderComponent = async (options: IRenderAndPerformActionsOptions = {}) =>
 	}
 };
 
+const mockID = "mock-random-id";
+jest.mock("src/utils/id-helper.ts", () => ({
+	generateRandomId: jest.fn(() => mockID),
+}));
+
```

Comment:
instead of mocking the entire file, you can do a `spyOn` on that function in that unit test 
```
import * as IdHelper from "../../../../utils/id-helper";

const mockID = "mock-random-id";
jest.spyOn(IdHelper, "generateRandomId").mockReturnValue(mockID);
```

Reply:
applied